experiments/operational_domain/operational_domain_3d_bestagon.cpp

```diff
@@ -0,0 +1,208 @@
+//
+// Created by marcel on 22.07.23.
```

Comment:
@marcelwa isn't this date outdated?

Reply:
Of course, sorry. I adjusted the date but not the year :sweat_smile: